templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -173,6 +173,13 @@ contents:
           connection.autoconnect-priority 100 802-3-ethernet.mtu ${iface_mtu} ${extra_phys_args}
       fi
 
+      # Update all connections that has master to the new device
+      conns=$(nmcli --fields UUID conn show | sed 1d)
+      while IFS= read -r conn; do
+        new_device=$(nmcli --get-values connection.interface-name conn show ${conn})
+        nmcli conn mod ${conn} connection.master=${new_device}
```

Comment:
this doesn't seem right. It sets the master property of a profile to its own interface-name.

Reply:
hmm I though the above will give the same effect of doing 
sed -i s/master=${old_conn}/master=${new_device}/ $NM_CONN_PATH/*.nmconnection

Reply:
ah, I think it only would have the same effect,if the previous `master=` as set to `${old_conn}`.

So you would need something like
```
   nmcli -g connection.master connection show uuid "$conn" | grep -q -F "$old_conn" || continue
```

Reply:
nmcli -g connection.master connection show 
Error: invalid field 'connection.master'; allowed fields: NAME,UUID,TYPE,TIMESTAMP,TIMESTAMP-REAL,AUTOCONNECT,AUTOCONNECT-PRIORITY,READONLY,DBUS-PATH,ACTIVE,DEVICE,STATE,ACTIVE-PATH,SLAVE,FILENAME.

Reply:
i said `nmcli -g connection.master connection show uuid "$conn"` :)